src/utils/getImportPath.ts

```diff
@@ -14,3 +14,17 @@ export function getImportPath(from: string, to: string) {
 
   return `${dir}/${name}`;
 }
+
+export function areImportPathsEqualIgnoringExtension(
+  path1: string,
+  path2: string
+): boolean {
+  return relative(normalizePath(path1), normalizePath(path2)) === "";
+}
+
+const normalizePath = (path: string) => {
+  const { dir, name } = parse(normalize(path));
+  // Ensure directory path ends with a separator for consistency
+  const dirWithSeparator = dir ? `${dir}/` : "";
```

Comment:
does this slash need to take platform specifics into account ( e.g. maybe on Windows it should be a "\")?

Reply:
This is only meant to handle import paths, not absolute ones.

Can import paths have backslashes? I really don't know 🤷‍♂️

Reply:
how about using `path.sep` instead of a hardcoded `/`?
see https://nodejs.org/api/path.html#pathsep

Reply:
or even better, do not manually build the path, but instead use `path.join`: 

```tsx
return join(dir, name);
```

Reply:
Done ✅